src/thunderbots/software/util/parameter/parameter.h

```diff
@@ -6,128 +6,231 @@
 #include <string>
 #include <vector>
 
+//messages for dynamic_reconfigure
+#include <dynamic_reconfigure/BoolParameter.h>
+#include <dynamic_reconfigure/DoubleParameter.h>
+#include <dynamic_reconfigure/IntParameter.h>
+#include <dynamic_reconfigure/StrParameter.h>
+
+//message that contains arrays of the xmlrpc types for reconf
+#include <dynamic_reconfigure/Config.h>
+
+//message for the reconfigure srv, takes a config msg
+#include <dynamic_reconfigure/Reconfigure.h>
+
+
 /**
  * This class defines a dynamic parameter, meaning the parameter
  * value can be changed during runtime. Although the class is templated, it is only meant
  * to support the types also supported by the ROS Parameter Server.
  *
  * See http://wiki.ros.org/Parameter%20Server for the list of types
  *
- * In our codebase, we support bool, int32_t, double, string, and lists (aka vectors)
- * of the previous 4 types
- */
+ * In our codebase, we support bool, int32_t, double, and strings
```

Comment:
Is there a universal place that we could point people to that indicates what types we support (so that this comment doesn't get out-of-sync if we add support for a new type in the future)?

Reply:
added it to the readme of param server